packages/api/package.json

```diff
@@ -8,7 +8,7 @@
         "react": "^17.0.2"
     },
     "devDependencies": {
-        "@deriv/api-types": "^1.0.54",
+        "@deriv/api-types": "^1.0.11",
```

Comment:
same here?

Reply:
Because it is failing some of the test cases in codecov and it doesn't have that specific call unless you update it.